packages/auth/src/features/authenticate/handler.test.ts

```diff
@@ -71,7 +71,19 @@ describe('authenticate handler receives a request', () => {
       expect(res.statusCode).toBe(403)
     })
     it('generates a mobile verification code and sends it to notification gateway', async () => {
-      server = await createServerWithEnvironment({ NODE_ENV: 'production' })
+      server = await createServerWithEnvironment({
+        NODE_ENV: 'production',
+        AUTH_HOST: '0.0.0.0',
+        AUTH_PORT: '4040',
+        CLIENT_APP_URL: 'http://localhost:3000/',
+        COUNTRY_CONFIG_URL: 'http://localhost:3040/',
+        DOMAIN: '*',
+        LOGIN_URL: 'http://localhost:3020/',
+        METRICS_URL: 'http://localhost:1050',
+        NOTIFICATION_SERVICE_URL: 'http://localhost:2020/',
+        REDIS_HOST: 'localhost',
+        USER_MANAGEMENT_URL: 'http://localhost:3030/'
```

Comment:
@naftis how about we take the variables from AUTH_HOST to USER_MANAGEMENT_URL to inside `createServerWithEnvironment`?

Reply:
I would have wanted to avoid it as we have this:

https://github.com/opencrvs/opencrvs-core/blob/a33c55c407a46bd54e0621c2a980fba966812566/packages/auth/src/features/resend/handler.test.ts#L84

to keep devDefaults only in one place.

I'll create a separate createProductionEnvironment.